gulp-core-build-sass/src/SassTask.ts

```diff
@@ -16,18 +18,22 @@ export interface ISassTaskConfig {
   postamble?: string;
   /** An array of glob patterns for locating SASS files. */
   sassMatch?: string[];
-  /** 
-   * If this option is specified, files ending with .module.scss extension will
+  /**
+   * If this option is specified, ALL files will be treated as a module.scss and will
```

Comment:
Right, but we do this anyway as long as the file is named .module.css, regardless of whether this flag is on or not.